src/axolotl/loaders/adapter.py

```diff
@@ -206,7 +206,7 @@ def load_llama_adapter(
         peft_model = PeftModel.from_pretrained(
             model,
             cfg.lora_model_dir,
-            torch_dtype=torch.float16,
+            dtype=torch.float16,
```

Comment:
Can you double check if this error is raised in PeftModel? I only see it in PretrainedModel.from_pretrained . https://github.com/huggingface/transformers/blob/f64354e89a55a456e024b301468da2dbb10e11b3/src/transformers/modeling_utils.py#L2320-L2321

For PeftModel, I see that it already loads the proper `dtype` attribute but not clear if there's BC handling for torch_dtype https://github.com/huggingface/peft/blob/50329a713899cc4f963e26142b1ca688a6166882/src/peft/peft_model.py#L1464-L1465